.github/instructions/xaml.instructions.md

```diff
@@ -0,0 +1,59 @@
+---
+description: 'Guidelines for XAML development in UWP applications'
+applyTo: '**/*.xaml'
+---
+
+# XAML Development
+
+## XAML Best Practices
+
+- Use x:Bind over Binding for better performance and compile-time type checking.
+- Specify binding Mode explicitly (OneWay, TwoWay, OneTime) for clarity.
+- Use StaticResource for static resources like brushes and styles.
+- Use ThemeResource for theme-aware resources that respond to theme changes.
+- Name controls that are referenced in code-behind with x:Name.
+- Use resource dictionaries for reusable styles and templates.
+- Minimize XAML code-behind; prefer data binding and view models for logic.
+- Keep XAML files focused and readable; extract complex templates into separate resource dictionaries.
+
+## Data Binding
+
+- Prefer x:Bind for better performance and compile-time validation.
+- Use FallbackValue and TargetNullValue to handle missing or null data gracefully.
+- Bind to view model properties rather than directly to models.
+- Use converters for data transformations; keep them in the Converters directory.
```

Comment:
Probably should clarify that it should use x:Bind functions over `IValueConverter`